src/Microsoft.Framework.ApplicationHost2/project.json

```diff
@@ -7,7 +7,7 @@
         "Newtonsoft.Json": "6.0.6",
         "Microsoft.Framework.Runtime.Roslyn.Interfaces": "1.0.0-*",
         "Microsoft.Framework.Runtime.Interfaces": "1.0.0-*",
-        "Microsoft.Framework.NotNullAttribute.Internal": { "type": "build", "version": "1.0.0-*" }
+        "Microsoft.Framework.NotNullAttribute.Internal": { "type": "build", "version": "1.0.0-*" },
```

Comment:
No, but it is valid JSON.NET :). Still, I'll fix it. It's an artifact of my initial solution which was to add DI.Interfaces :)